classes/PodsAdmin.php

```diff
@@ -1311,6 +1311,12 @@ public function admin_setup_edit_proto( $duplicate, $obj ) {
 				'meta'    => _x( 'Meta', 'storage type label', 'pods' ),
 				'table'   => _x( 'Table', 'storage type label', 'pods' ),
 			],
+			'wp_locale'      => $GLOBALS['wp_locale'],
+			'currencies'     => PodsField_Currency::$currencies,
+			'datetime'       => [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 82a4a5cbca35afa01c71b6a136d821e1</sub>